packages/jaeger-ui/src/components/TracePage/TraceStatistics/generateDropdownValue.tsx

```diff
@@ -41,15 +45,13 @@ function getValueTagIsPicked(tableValue: ITableSpan[], trace: Trace, nameSelecto
   }
   availableTags = [...new Set(availableTags)];
 
-  const tags = _(availableTags).map('tags').flatten().value();
-  let tagKeys = _(tags).map('key').uniq().value();
-  tagKeys = _.filter(tagKeys, function calc(o) {
-    return o !== nameSelectorTitle;
-  });
+  const tags = _map(availableTags, 'tags');
+  let spansWithFilterTag = _uniq(_map(_flatten(tags), 'key'));
+  spansWithFilterTag = _filter(spansWithFilterTag, o => o !== nameSelectorTitle);
```

Comment:
this collection does not contain spans, now it's even more confusing. I was referring to L46 and above, where the objects in question are spans, but the collection is called availableTags